aspnetcore/index.yml

```diff
@@ -233,6 +233,24 @@ conceptualContent:
         - url: https://docs.microsoft.com/aspnet/overview
           itemType: concept
           text: "ASP.NET 4.x"
+      # Card
+    - title: ASP.NET Core video tutorials
```

Comment:
Observation: This card seems out of place. It doesn't fit the theme of "what you can build with ASP.NET Core" in the rest of this section. Is there a better place to put it?

Reply:
I'll admit it wasn't a great fit.  The videos are centered on building, but of course it is a collection of different app types for one card and the rest of the cards are one app type per card.   If I created a section for it, it would really be odd, and take up a lot extra space.